tests/v2/validation/charts/k8srequiredlabels.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
```

Comment:
Can we move this file to a resources directory?

Reply:
Like this? `tests/v2/validation/charts/resources/k8srequiredlabels.yaml`

Reply:
That should be fine - I would also prefix the file name with `opa` to signify that's what it's used for.

Reply:
Moved it to 'tests/v2/validation/charts/resources/opa-k8srequiredlabels.yaml'